CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Unreleased Changes
-## Suggested release: v4.1.1
+(none)
```

Comment:
Remove the (none) and unreleased changes, we don't want it on our official gem release!